.travis.yml

```diff
@@ -33,12 +33,16 @@ install:
   # Install all Python dependencies
   - conda install -q --yes python=$TRAVIS_PYTHON_VERSION mkl --file req-travis-conda.txt
   - pip install -q -r req-travis-pip.txt
+  - pip install -q git+git://github.com/mila-udem/blocks-extras.git 
+  - conda install -q --yes bokeh
+  - conda install -q --yes pandas
```

Comment:
From what I remember from doing something similar on blocks-examples, the travis build fails without pandas. I think it has something to do with bokeh but I'll look into it again in a few hours.